dockerfiles/nginx/proxito.conf.template

```diff
@@ -88,6 +88,18 @@ server {
         proxy_hide_header Content-Security-Policy;
         set $content_security_policy $upstream_http_content_security_policy;
         add_header Content-Security-Policy $content_security_policy always;
+
+        # This header allows us to decide whether or not inject the script at CloudFlare level
+        # Now, I'm injecting it in all the NGINX responses because `sub_filter` is not allowed inside an `if` statement.
+        set $rtd_hosting_integrations $upstream_http_x_rtd_hosting_integrations;
+        add_header X-RTD-Hosting-Integrations $rtd_hosting_integrations always;
```

Comment:
I think we need this on `ops` as well, when we deploy this.

Reply:
Good point. Opened at https://github.com/readthedocs/readthedocs-ops/pull/1319